### PR TITLE
Russian Peasant Algorithm in Java

### PR DESCRIPTION
**Pull request by ritvij14**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)

### Description
Russian Peasant multiplication algorithm in Java.
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Issue #978 
(Write your answer here.)

### CODE REVIEW

**Comment by ritvij14** [comment]
@iamrajiv @plazzy99 @Kajol-Kumari @siddharth25pandey please review this.

File: Java/Algorithms/RussianPeasant.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import java.util.*;
```
Comment (iamrajiv) [inline Java/Algorithms/RussianPeasant.java:9]: @ritvij14 Added description about Russian Peasant Algorithm.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: Java/Algorithms/RussianPeasant.java (hunk 1 above)
Comment (ritvij14) [reply Java/Algorithms/RussianPeasant.java:9]: @iamrajiv done.

**Review by iamrajiv — APPROVED** [review]

**Review by vatsalkesarwani12 — APPROVED** [review]

**Review by Kajol-Kumari — APPROVED** [review]
Well formatted and Looks good to Go! :+1:

**Comment by ritvij14** [comment]
@iamrajiv sir here also, 3 approvals done, please merge

**Pull request merged**